### DISCUSSION TITLE
Updating the online documentation

### DISCUSSION THREAD

**Discussion by ferozsalam** [jertel/elastalert2#56] [Ideas]
The current documentation site that's available (at https://elastalert.readthedocs.io/) is quite outdated and contains numerous errors. As we're also continuing to make updates to the package, it would be useful to be able to point users to a new documentation site that better reflects the current state of the software.

For ease of transition I suggest that we continue to use Read The Docs, perhaps setting up the new site at `elastalert-alt.readthedocs.io`.

I am happy to set this up myself and update the links to the documentation in various places, but wanted to post here first to see if anyone has any input or suggestions before I do so.

**Comment by jertel** [comment]
I have RTD building at this location: https://elastalert2.readthedocs.io/en/latest/ but haven't updated links to point to it. I was hoping to see some activity on the original project after the holidays. Since that's not happened it's probably safe to update the links.

**Reply by ferozsalam** [reply]
Nice! OK - I'll update the links and put in a PR in a bit.
